### ISSUE TITLE
Add support for landscape activity

### ISSUE THREAD

**Issue by raviraj16** [siralam/CameraViewPlus#1]
Please add support for the activity whose orientation is declared landscape in manifest.
Currently preview is showing different with wrong orientation and sometimes the preview is getting stretched.

**Comment by siralam** [comment]
@raviraj16 I am sorry, but the whole idea and design of this library assumes portrait activity. I have no plan to support for landscape activity at this moment.

I recommend [fotoapparat](https://github.com/Fotoapparat/Fotoapparat) if you need a CameraView that supports landscape activity.

**Comment by raviraj16** [comment]
@siralam Thanks for your early response, I have already tried [fotoapparat](https://github.com/Fotoapparat/Fotoapparat) and [CameraKit ](https://github.com/CameraKit/camerakit-android) libraries but they have also the same bug and it is not fixed yet.

**Comment by siralam** [comment]
@raviraj16 How about this [CameraView](https://github.com/natario1/CameraView)? I did encounter one distortion problem in a 4.4.1 device so I did not go ahead to use it; but it might suit your need.

If not... I guess no luck then :(  
Will you consider making a "fake" landscape activity? For example, my Sony built-in camera app use portrait activity but rotate their layouts (yes, including words!) by detecting device orientation using sensors.
